docs/content/docs/build/dds.md

```diff
@@ -179,18 +183,33 @@ This results in someone's changes being "lost" from a user's perspective. This m
 However, if your scenario requires users to edit individual properties of the shape, then the SharedMap LWW merge
 strategy probably won't give you the behavior you want.
 
-However, you could address this problem by storing individual shape properties in `SharedMap` keys. Instead of storing a
+However, you could address this problem in different ways depending on which version of Fluid Framework you are using.
+
+In version 1.0, store individual shape properties in `SharedMap` keys. Instead of storing a
 JSON object with all the data, your code can break it apart and store the length in one `SharedMap` key, the width in another,
 etc. With this data model, users can change individual properties of the shape without overwriting other users' changes.
 
 You likely have more than one shape in your data model, so you could create a `SharedMap` object to store all the shapes, then
 store the `SharedMaps` representing each shape within that parent `SharedMap` object.
 
+In version 2.0, there's a similar, but better, way. Store individual shape properties in a map node of a `SharedTree`. Your code can store the length in one key of the map node, the width in another, etc. Again, users can change individual properties of the shape without overwriting other users' changes.
+
+When you have more than one shape in your data model, you could create a *list* node in the `SharedTree`, with child map nodes to store all the shapes.
+
 ### Key-value data
 
 These DDSes are used for storing key-value data. They are all optimistic and use a last-writer-wins merge policy.
 
 *   [SharedMap][] -- a basic key-value distributed data structure.
+*   Map nodes in a [SharedTree][] -- a hierarchical data structure with three kinds of complex nodes; maps (similar to [SharedMap][]), array-like lists, and JavaScript objects. There are also several kinds of leaf nodes, including boolean, string, number, null, and [Fluid handles]({{< relref "handles.md" >}}).
+
+### Array-like data
```

Comment:
We are renaming List to Array to be more consistent with object and map and to acknowledge that we have to explain to everyone that a List is basically an array, with all the read-only array methods, but with a different write API better suited to collab - it has move, insert, and remove instead of push, pop, shift, unshift etc.